enigma-tools-u/src/web3_utils/raw_transaction.rs

```diff
@@ -0,0 +1,156 @@
+use ethereum_types::{H160, U256};
+use rlp::RlpStream;
+//use secp256k1::key::SecretKey;
+//use secp256k1::Message;
+//use secp256k1::Secp256k1;
+use tiny_keccak::keccak256;
+
+/// Description of a Transaction, pending or in the chain.
+#[derive(Debug, Default, Clone, PartialEq, Deserialize, Serialize)]
+pub struct RawTransaction {
+    /// Nonce
```

Comment:
i have no idea if anything in this module is correct, but please remove all commented out code